app/workers/retirement_worker.rb

```diff
@@ -15,8 +15,15 @@ def perform(count_id)
 
   def deactivate_workflow!(workflow)
     if workflow.finished?
-      workflow.active = false
-      workflow.save!
+      ignore_optimistic_lock { workflow.update_attribute(:active, false) }
     end
   end
+
+  private
+
+  def ignore_optimistic_lock
+    ActiveRecord::Base.lock_optimistically = false
```

Comment:
So I'm not 100% this will be thread safe. Since sidekiq runs in one process and executes a bunch of threads won't this disable optimistic locking for anything running in a concurrent sidekiq thread?